nautobot/docs/docker/index.md

```diff
@@ -0,0 +1,159 @@
+
+```no-highlight
+$ docker run --name nautobot -v /local/path/to/custom/nautobot_config.py:/opt/nautobot/nautobot_config.py ghcr.io/nautobot/nautobot
+```
+
+Or if you are using docker-compose:
+
+```yaml
+services:
+  nautobot:
+    image: "ghcr.io/nautobot/nautobot"
+    volumes:
+      - /local/path/to/custom/nautobot_config.py:/opt/nautobot/nautobot_config.py:ro
+```
+
+### uWSGI
+
+The docker container uses [uWSGI](https://uwsgi-docs.readthedocs.io/) to serve Nautobot.  A default configuration is [provided](/docker/uwsgi.ini), and can be overridden by injecting a new `uwsgi.ini` file at `/opt/nautobot/uwsgi.ini`.  There are a couple of environment variables provided to override some uWSGI defaults:
+
+#### `NAUTOBOT_UWSGI_LISTEN`
+
+Default: `128`
+
+The socket listen queue size of uWSGI.  In production environments it is recommended to increase this value to 1024 or higher, however depending on your platform, this may require additional kernel parameter settings, please see [the uWSGI documentation](https://uwsgi-docs.readthedocs.io/en/latest/articles/TheArtOfGracefulReloading.html#the-listen-queue) for more information.
+
+Please see the [official uWSGI documentation on `listen`](https://uwsgi-docs.readthedocs.io/en/latest/Options.html?highlight=listen#listen) for more information.
+
+---
+
+#### `NAUTOBOT_UWSGI_PROCESSES`
+
+Default: `3`
+
+The number of worker processes uWSGI will spawn.
+
+Please see the [official uWSGI documentation on `processes`](https://uwsgi-docs.readthedocs.io/en/latest/Options.html?highlight=processes#processes) for more information.
+
+---
+
+### SSL
+
+Self signed SSL certificates are included by default with the container.  For a production deployment you should utilize your own signed certificates, these can be injected into the container at runtime using [docker volumes](https://docs.docker.com/storage/volumes/).  The public certificate should be placed at `/opt/nautobot/nautobot.crt` and the private key should be at `/opt/nautobot/nautobot.key`.  Using a `docker run` these can be injected using the `-v` parameter:
+
+```no-highlight
+$ docker run --name nautobot -v /local/path/to/custom/nautobot.crt:/opt/nautobot/nautobot.crt -v /local/path/to/custom/nautobot.key:/opt/nautobot/nautobot.key ghcr.io/nautobot/nautobot
+```
+
+Or if you are using `docker-compose`:
+
+```yaml
+services:
+  nautobot:
+    image: "ghcr.io/nautobot/nautobot"
+    volumes:
+      - /local/path/to/custom/nautobot.crt:/opt/nautobot/nautobot.crt:ro
+      - /local/path/to/custom/nautobot.key:/opt/nautobot/nautobot.key:ro
+```
+
+### Nautobot Plugins
+
+At this time adding Nautobot plugins to the existing Docker image is not supported, however, you can use the Nautobot image as the base within your `Dockerfile` to install your own plugins, here is an example dockerfile:
+
+```dockerfile
+FROM ghcr.io/nautobot/nautobot
+
+RUN pip install nautobot-chatops
+```
```

Comment:
You'd also need to be sure to replace `nautobot_config.py` with one that has the appropriate `PLUGINS` and/or `PLUGIN_CONFIG` values to actually use the plugin.